### PR TITLE
[IMP] l10n_nl: translation

### PR DESCRIPTION
**Pull request by malb-odoo**
All the localisation should be written in english and then translated to the native language of the country. In this case, the Dutch localisation was written is Dutch, this PR translate all the module in english and add the corresponding PO file in Dutch.

Task-id: 3115909

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_nl/data/account_account_tag.xml [hunk 1]
```diff
@@ -4,138 +4,137 @@
...
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_2" model="account.account.tag">
-            <field name="name">Huisvestingskosten</field>
+            <field name="name">Accommodation costs</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_3" model="account.account.tag">
-            <field name="name">Financiële vaste activa</field>
+            <field name="name">Immobilisations financières</field>
```
Comment (qdp-odoo) [inline addons/l10n_nl/data/account_account_tag.xml:15]:
```suggestion
            <field name="name">Financial fixed assets</field>
```

**Comment by qdp-odoo** [comment]
looks good for me, we might wait for the branch of William to be ready, though.

Can you describe how you produced the .po file to keep the existing translations?
Thanks

**Review by qdp-odoo — CHANGES_REQUESTED** [review]

**Comment by malb-odoo** [comment]
> looks good for me, we might wait for the branch of William to be ready, though.
> 
> Can you describe how you produced the .po file to keep the existing translations? Thanks

Either wait for William branch or apply a script afterward, either way is fine for me. 
To get the .po file, you can add the desire language here is Dutch/Nederlands, then go to setting while being in debug mode you can export translation. If you pick a language a Po file will be created, if not a Pot file will be created. Normally if the language is the same as the old PO file, all the translation will be kept. I hope it answer the question :smile:

**Review by smetl — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**